### PR TITLE
Fix/carousel arrows

### PR DESCRIPTION
**Pull request by igorbrasileiro**
## What's the purpose of this pull request?
Remove hardcoded background-color and color from the carousel arrows.

## How to test it?
Use a carousel component, change the arrows background-color or color, observe it change.

### CODE REVIEW

File: packages/store-ui/src/Carousel/ArrowLeft.tsx [hunk 1]
```diff
@@ -6,12 +6,7 @@ import SliderArrowLeft from '../Slider/ArrowLeft'
 type Props = ComponentProps<typeof SliderArrowLeft>
 
 const CarouselArrowLeft: FC<Props> = (props) => (
-  <SliderArrowLeft
-    backgroundColor="transparent"
-    color="black"
-    aria-label="Previous Carousel Image"
-    {...props}
-  />
+  <SliderArrowLeft aria-label="Previous Carousel Image" {...props} />
```
Comment (emersonlaurentino) [inline packages/store-ui/src/Carousel/ArrowLeft.tsx:9]: put aria-label to come from the prop too

File: packages/store-ui/src/Carousel/ArrowRight.tsx [hunk 2]
```diff
@@ -6,12 +6,7 @@ import SliderArrowRight from '../Slider/ArrowRight'
 type Props = ComponentProps<typeof SliderArrowRight>
 
 const CarouselArrowRight: FC<Props> = (props) => (
-  <SliderArrowRight
-    backgroundColor="transparent"
-    color="black"
-    aria-label="Next Carousel Image"
-    {...props}
-  />
+  <SliderArrowRight aria-label="Next Carousel Image" {...props} />
```
Comment (emersonlaurentino) [inline packages/store-ui/src/Carousel/ArrowRight.tsx:9]: put aria-label to come from the prop too

File: packages/store-ui/src/Carousel/ArrowLeft.tsx (hunk 1 above)
Comment (igorbrasileiro) [reply packages/store-ui/src/Carousel/ArrowLeft.tsx:9]: Default props with `-` character can't be passed to `aria-label` prop.

Comment (igorbrasileiro) [reply packages/store-ui/src/Carousel/ArrowLeft.tsx:9]: Also, if aria-label is passed through props, it will override.

**Review by emersonlaurentino — APPROVED** [review]

**Pull request merged**